lib/kapost_deploy/plugins/notify_honeybadger_after_promote.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require "kapost_deploy/heroku/app_releases"
+
+module KapostDeploy
+  module Plugins
+    # After-promotion plugin to notify honeybadger after a promotion is complete.
+    # Honeybadger.io can be configured to clear all honeybadgers upon this deploy notification.
+    class NotifyHoneyBadgerAfterPromote
```

Comment:
The reason I used it is because the notification only occurs after promotion so it's clear when the notification occurs.  The plugin is hard-coded to call the hb code in `after` so you would need to modify the plugin code or make it configurable but I don't think there is a use case for that.  However, I could add `Deploy` in the name to better match the rake task name.

Further thoughts?